voting/processor/src/orbs.js

```diff
@@ -101,8 +101,18 @@ class Orbs {
     }
 }
 
-function create(url, vChainId, name) {
-    return new Orbs(url, vChainId, name)
+async function create(urlsString, vChainId, name) {
+    let urls = urlsString.split(',');
+    for (let i = 0;i < urls.length;i++) {
+        try {
+            let orbs = new Orbs(urls[i], vChainId, name);
+            await orbs.getNumberOfElections();
```

Comment:
why is this called here? is this a roundabout way to initialize some object state?